From ca6578cc38adc747f79f75c3dc56c96118a87d6a Mon Sep 17 00:00:00 2001
From: Peng Wu <wupeng58@huawei.com>
Date: Tue, 12 Apr 2022 02:35:42 +0000
Subject: [PATCH 3546/4585] [Huawei] mm: Add reliable_nr_page for accounting
 reliable memory

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187270

------------------------------------------

Adding an variable in mm_struct for accouting the amount of reliable memory
allocated by the reliable user tasks.

Signed-off-by: Peng Wu <wupeng58@huawei.com>
---
 include/linux/mm_types.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/linux/mm_types.h b/include/linux/mm_types.h
index 1c22e294f083..15ff1e20f5ca 100644
--- a/include/linux/mm_types.h
+++ b/include/linux/mm_types.h
@@ -587,7 +587,8 @@ struct mm_struct {
 #endif
 
 #ifdef CONFIG_MEMORY_RELIABLE
-		atomic_long_t reserve_0;
+		/* total used reliable pages */
+		KABI_RENAME(atomic_long_t reserve_0, atomic_long_t reliable_nr_page);
 #endif
 	} __randomize_layout;
 
-- 
2.17.1

